lib/src/http/managers/message_manager.dart

```diff
@@ -230,8 +231,12 @@ class MessageManager extends Manager<Message> {
     return MessageReference(
       manager: this,
       messageId: maybeParse(raw['message_id'], Snowflake.parse),
+
+      /// TODO: Apparently, channel_id is nullable.
```

Comment:
Going to close this PR since I've just implemented it anyway, but FWIW the docs say that this field isn't nullable when you _receive_ it from the API:

> `channel_id` is optional when creating a reply, but will always be present when receiving an event/response that includes this data model. **Required for forwards.**